airflow/www/templates/airflow/dags.html

```diff
@@ -203,7 +203,7 @@ <h2>{{ page_title }}</h2>
           </th>
           <th style="width:180px;">{{ sortable_column("Next Run", "next_dagrun") }}
             <span class="material-icons text-muted js-tooltip" aria-hidden="true"
-                  title="Expected Date/Time of the next Dag Run.">info</span>
+                  title="Expected Date/Time of the next Dag Run, or for Dataset triggered DAGs, how many Datasets have been updated since the last Dag Run.">info</span>
```

Comment:
The capitalization of this sentence is a little inconsistent, even internally ("DAG" vs. "Dag Run").

```suggestion
title="Expected date/time of the next Dag Run, or for dataset triggered DAGs, how many datasets have been updated since the last DAG Run.">info</span>
```